<?xml version="1.0"?>
<launch>

  <!-- <node pkg="image_transport" type="republish" name="decompress_node" output="screen" args="raw in:=/zed2/zed_node/depth/depth_registered/compressed out:=test_depth"/> -->
  <!-- relay to avoid subscribing multiple times to rgb image -->
  <node name="republish_depth" type="republish" pkg="image_transport" args="compressed in:=/zed2/zed_node/rgb/image_rect_color/compressed raw out:=/zed_depth" />

  <!--node pkg="nodelet" type="nodelet" name="nodelet1"
        args="load depth_image_proc/point_cloud_xyz nodelet_manager">
    <remap from="camera_info" to="/zed2/zed_node/rgb/camera_info"/>
    <remap from="image_rect" to="/zed2/zed_node/depth/depth_registered"/>
    <remap from="points" to="/camera/depth/points"/>
  </node-->

  <!-- <node pkg="nodelet" type="nodelet" name="nodelet1"
        args="load depth_image_proc/point_cloud_xyzrgb nodelet_manager">
    <remap from="rgb/camera_info" to="/zed2/zed_node/rgb/camera_info"/>
    <remap from="rgb/image_rect_color" to="/zed2/zed_node/rgb/image_rect_color"/>
    <remap from="depth_registered/image_rect" to="/zed2/zed_node/depth/depth_registered"/>
    <remap from="depth_registered/points" to="/camera/depth/points"/>
  </node> -->

</launch>